src/main.rs

```diff
@@ -286,18 +271,28 @@ fn main() {
                     Err(e) => error!("Failed to clean {:?}: {}", project_path, e),
                 };
             }
-
-            return;
-        }
-
-        for project_path in &paths {
-            match remove_older_than(project_path, &keep_duration, dry_run) {
-                Ok(cleaned_amount) if dry_run => {
-                    info!("Would clean: {}", format_bytes(cleaned_amount))
-                }
-                Ok(cleaned_amount) => info!("Cleaned {}", format_bytes(cleaned_amount)),
-                Err(e) => error!("Failed to clean {:?}: {}", project_path, e),
+        } else {
+            let keep_duration = if matches.is_present("file") {
+                let ts = Timestamp::load(path.as_path()).expect("Failed to load timestamp file");
+                Duration::from(ts)
+            } else {
+                let days_to_keep: u64 = matches
+                  .value_of("time")
+                  .expect("--time argument missing")
```

Comment:
As far as I can tell, the `expect("--time argument missing")` can be replaced with an `unwrap()`, as `matches.value_of("time")` should always be set at this point in execution. I left it in because it doesn't really do any harm being there, and I could be wrong about it being unreachable.